docs/website/root/mithril/threat-model.md

```diff
@@ -0,0 +1,318 @@
+
+#### Era activation 
+
+The current and next (if any) eras are announced on-chain with an era activation marker (see ADR https://mithril.network/doc/adr/4#era-activation-marker).
+
+* **confidentiality**: No (they actually need to be public)
+* **availability**: Yes (era markers are currently stored on-chain)
+* **integrity**: Yes (tampering the era markers could lead to Mithril network not being able to create multi-signatures)
+
+#### Era verification vey
+
+Era verification key is stored in [GitHub](https://raw.githubusercontent.com/input-output-hk/mithril/main/mithril-infra/configuration/release-mainnet/era.vkey) and only modifiable through an approved merged PR.
+
+* **availability**: Yes (it's needed to verify a whole chain of certificate)
+* **confidentiality**: No
+* **integrity**: No (integrity is inherent to the protocol)
+
+#### Era signing key
+
+The corresponding signing key is stored in IOG's VaultWarden, and used only when a new era is announced or activated.
+
+* **availability**: No 
+* **confidentiality**: Yes
+* **integrity**: Yes (?)
+
+### Client-side only assets
+
+These are the assets that are relevant only when downloading and verifying certificates and full-node snapshots. 
+
+:::info To do
+
+Verification process being trustless, does it really make sense to include those in the Threat Model?
+
+:::
+
+#### Mithril certificate verification process
+
+Mithril clients download snapshots and verify associated certificates using the mithril-client library, either from a CLI tool or [embedded in a browser](https://mithril.network/explorer/) 
+
+* **availability**: Yes (?)
+* **confidentiality**: No
+* **integrity**: No (the snapshots and certificates are assumed to be secure, integrity is inherent to the protocol)
+
+#### Mithril snapshots & certificates
+
+Mithril certificates are produced by the aggregator from the individual signatures. Mithril certificates' security rests upon a chain of trust ultimately pointing at the Genesis certificate. Mithril aggregator maintains snapshots and certificates (aggregated multi-signatures) to be served to clients
+
+* **availability**: No (Snapshots and certificates are just a fallback, a cardano-node client can always retrieve the infromation from the cardano network itself, albeit much more slowly)
+* **confidentiality**: No
+* **integrity**: No (the snapshots and certificates are assumed to be secure, integrity is inherent to the protocol)
+
+#### Mithril Genesis Verification Key
+
+Mithril genesis verification key is stored in [GitHub](https://github.com/input-output-hk/mithril/blob/main/mithril-infra/configuration/release-mainnet/genesis.vkey) and only modifiable through an approved merged PR
+
+* **availability**: Yes (it's needed to verify a whole chain of certificate)
+* **confidentiality**: No
+* **integrity**: No (integrity is inherent to the protocol)
+
+#### Mithril Genesis Signing Key
```

Comment:
Is this a "client-side only asset" (section this is in)? It sounds quite essential to the whole signing process?

Reply:
This is not a client-only asset: I moved it upper to the correct section :+1: